packages/salesforcedx-vscode-apex-replay-debugger/src/messages/i18n.ts

```diff
@@ -62,5 +62,12 @@ export const messages = {
     'Retrieved line breakpoint info from language server',
   language_client_not_ready:
     'Unable to retrieve breakpoint info from language server, language server is not ready',
-  unable_to_retrieve_org_info: 'Unable to retrieve OrgInfo'
+  unable_to_retrieve_org_info: 'Unable to retrieve OrgInfo',
```

Comment:
cc @ekapner

Reply:
@lcampos @rcoringrato-sfdc - where does this message show up? Would like to rewrite it; potentially provide information to recover.